cluster/gce/gci/configure-helper.sh

```diff
@@ -1575,6 +1613,9 @@ if [[ "${KUBERNETES_MASTER:-}" == "true" ]]; then
   create-master-auth
   create-master-kubelet-auth
   create-master-etcd-auth
+  if [[ "${ENABLE_APISERVER_ADVANCED_AUDIT:-}" == "true" ]]; then
```

Comment:
is it possible to move this into the top of start-kube-apiserver where we setup kube-apiserver's log files?